astroplan/tests/test_constraints.py

```diff
@@ -206,7 +206,12 @@ def test_moon_illumination():
     is_constraint_met = [constraint(lco, None, times=time) for time in times]
     assert np.all(is_constraint_met == [False, True, True])
 
-
+    
+def test_Dark_Grey_Bright():
+    # TODO: write the test
+    pass
```

Comment:
Time to fill in this test. 

Don't forget to run the tests locally before making a pull request by running this from the top-level of the repository: 
```
python setup.py test
```
Run it with both py 2.7 and 3.5, which Anaconda makes [possible without infinite effort](http://stackoverflow.com/a/24415581/1340208).